src/OSPSuite.InstallationValidator/Views/MainView.cs

```diff
@@ -75,5 +68,20 @@ public void ValidationIsRunning(bool validationRunning)
       {
          OkEnabled = !HasError && !validationRunning;
       }
+
+      public void AppendText(string newText)
+      {
+         getInnerTextBox(memoEditLogMessages).AppendText(newText);
+      }
+
+      private TextBox getInnerTextBox(DevExpress.XtraEditors.TextEdit editor)
+      {
+         return editor?.Controls.OfType<TextBox>().FirstOrDefault();
```

Comment:
that's returning the TextBox that underlies the memo edit so that we can use the more efficient AppendText rather than concatenating strings by doing

memoEdit.Text += newText